pandas/tests/io/test_excel.py

```diff
@@ -119,6 +119,15 @@ def get_exceldf(self, basename, ext, *args, **kwds):
 class ReadingTestsBase(SharedItems):
     # This is based on ExcelWriterBase
 
+    @pytest.fixture(autouse=True, params=['xlrd', None])
```

Comment:
I agree with this decision.  In the future, in the interest of greater `pytest` idiom, it would be great (though not sure if possible) yet to break up this massive test file into a directory of `excel` tests, in which case we could put this fixture in the `conftest` file for `excel`.

But that's a ways off... 🙂